### PR TITLE
chore: update `release_created` to `releases_created` in README.md

### PR DESCRIPTION
**Pull request by cychiang**
I followed the instruction to create a release workflow and then I found a issue from the example in `README.md`. 

In the example workflow, the `outputs` from the `release-please-action` to determine a release is created or not should be `releases_created` and not `release_created`.

### CODE REVIEW

**Comment by jbottigliero** [comment]
Ran into this as well. The typo for the Output section was fixed in #626, but the examples were not updated.

File: README.md [hunk 1]
```diff
@@ -373,7 +373,7 @@ jobs:
         if: ${{ steps.release.outputs.releases_created }}
       # For root level packages: ${{ steps.release.outputs.package-a--tag_name }}
       - run: echo "Release ${{ steps.release.outputs['packages/package-a--tag_name'] }} created for package-a."
-        if: ${{ steps.release.outputs['packages/package-a--release_created'] }}
+        if: ${{ steps.release.outputs['packages/package-a--releases_created'] }}
```
Comment (jbottigliero) [inline README.md:376]: This instance might still be `release_created`: https://github.com/google-github-actions/release-please-action/blob/main/index.js#L234

Comment (cychiang) [reply README.md]: Thanks! Fix it, didn't notice it.

**Comment by chingor13** [comment]
Apologies for the confusion, there are 2 separate outputs: `releases_created` and `release_created`. The former (`releases_created`) will be true if any release was created (e.g. for a monorepo). The later `release_created` indicates if the "root" component had a release created or not.

The example use case is for a single component repo and so `release_created` should work (although `releases_created` should also work).

**Pull request closed without merging**